ci-infra/azure/variables.tf

```diff
@@ -0,0 +1,57 @@
+# Note: Using `ver` because `version` is a built-in variable.
+variable "ver" {
+  type        = number
+  description = "Monotonically increasing number to track version of infrastructure"
```

Comment:
I have mostly seen the usage of variable name `id` for such scenarios.  Seems a matter of preferences.